Frontend/src/Pages/Project/Create/ProjectDataStep.jsx

```diff
@@ -67,7 +67,7 @@ export default function ProjectDataStep({ image, setImage, nextStep, prevStep, h
                 <InputWithLabel defaultValue={ values.name } placeholder={t('title.placeholder')} type="text" label={t('title.label')} onChange={setTitle} />
                 <InputWithLabel defaultValue={ values.description } placeholder={t('description.placeholder')} type="textarea" label={t('description.label')} onChange={setDescription} />
                 <label className="input-with-label-label" htmlFor="videoType">
-                    {thumbnail('genreLabel')}
+                    {t('genreLabel')}
```

Comment:
![image](https://user-images.githubusercontent.com/41423449/211078158-60953207-1724-4b60-b494-b7c6e6776a41.png)
as t uses the prefix 'project.create.step2.form' I think the field 'genreLabel' doesn't correspond to anything

Reply:
![image](https://user-images.githubusercontent.com/41423449/211078791-c9e50a78-2d6b-409b-9b5e-37ee388736fe.png)
Result

Reply:
I suggest changing the 'genreLabel' to 'genre.label', should work properly